apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -6,6 +6,28 @@ import kotlin.test.Test
 
 class MapLikeToContainInAnyOrderCreatorSamples {
 
+    @Test
+    fun entry() {
+        expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "b")
+
+        fails {
+            expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entry(2 to "c")
+        }
+    }
+
+    @Test
+    fun entryKeyValue() {
+        expect(mapOf(1 to "apple", 2 to "banana")).toContain.inAnyOrder.entry(
+            KeyValue(2) { toStartWith("b") }
+        )
+
+        fails {
+            expect(mapOf(1 to "apple", 2 to "banana")).toContain.inAnyOrder.entry(
+                KeyValue(1) { toStartWith("b") } // fails because subject does not have this entry
```

Comment:
I just though, maybe we should be more explanatory
```suggestion
                KeyValue(1) { toStartWith("b") } // fails because the value ("apple") of key 1 (which exists in the subject) does not start with "b"
```
WDYT? Maybe you can also improve other explanations in this direction

Reply:
That makes a lot of sense. Thanks for this feedback!